sound/soc/intel/boards/sof_sdw.c

```diff
@@ -799,13 +799,6 @@ static void sof_sdw_check_ssid_quirk(const struct snd_soc_acpi_mach *mach)
 		sof_sdw_quirk = quirk_entry->value;
 }
 
-static struct snd_soc_dai_link_component platform_component[] = {
```

Comment:
Looks good @charleskeepax except for the typo in the commit title "don't both to set platform string"